### ISSUE TITLE
SAEHD trainer does not continue 

### ISSUE THREAD

**Issue by haydle360** [iperov/DeepFaceLab#5634]
THIS IS NOT TECH SUPPORT FOR NEWBIE FAKERS
POST ONLY ISSUES RELATED TO BUGS OR CODE

## Expected behavior

training begins and new training window opens

## Actual behavior

Running trainer.

Choose one of saved models, or enter a name to create a new model.
[r] : rename
[d] : delete

[0] : new - latest
 :
0
Loading new_SAEHD model...

Choose one or several GPU idxs (separated by comma).

[CPU] : CPU
  [0] : NVIDIA GeForce RTX 3060 Laptop GPU

[0] Which GPU indexes to choose:
0

Press enter in 2 seconds to override model settings.
[0] Autobackup every N hour ( 0..24 ?:help ) :
0
[n] Write preview history ( y/n ?:help ) :
n
[0] Target iteration :
0
[n] Flip SRC faces randomly ( y/n ?:help ) :
n
[y] Flip DST faces randomly ( y/n ?:help ) : n
[8] Batch_size ( ?:help ) : 4
4
[y] Masked training ( y/n ?:help ) :
y
[y] Eyes and mouth priority ( y/n ?:help ) :
y
[y] Uniform yaw distribution of samples ( y/n ?:help ) :
y
[y] Blur out mask ( y/n ?:help ) :
y
[y] Place models and optimizer on GPU ( y/n ?:help ) :
y
[y] Use AdaBelief optimizer? ( y/n ?:help ) : y
[y] Use learning rate dropout ( n/y/cpu ?:help ) :
y
[y] Enable random warp of samples ( y/n ?:help ) :
y
[0.0] Random hue/saturation/light intensity ( 0.0 .. 0.3 ?:help ) :
0.0
[0.0] GAN power ( 0.0 .. 5.0 ?:help ) :
0.0
[0.0] Face style power ( 0.0..100.0 ?:help ) :
0.0
[0.0] Background style power ( 0.0..100.0 ?:help ) :
0.0
[none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) : none
none
[n] Enable gradient clipping ( y/n ?:help ) :
n
[n] Enable pretraining mode ( y/n ?:help ) :
n
Initializing models: 100%|###############################################################| 5/5 [00:05<00:00,  1.01s/it]
Loaded 15115 packed faces from D:\Program Files\Deepface\DeepFaceLab - Live\workspace\data_src\aligned
Sort by yaw: 100%|##################################################################| 128/128 [00:00<00:00, 286.62it/s]
Loaded 63012 packed faces from D:\Program Files\Deepface\DeepFaceLab - Live\workspace\data_dst\aligned
Sort by yaw: 100%|###################################################################| 128/128 [00:02<00:00, 52.45it/s]
======================== Model Summary ========================
==                                                           ==
==            Model name: new_SAEHD                          ==
==                                                           ==
==     Current iteration: 1                                  ==
==                                                           ==
==---------------------- Model Options ----------------------==
==                                                           ==
==            resolution: 224                                ==
==             face_type: wf                                 ==
==     models_opt_on_gpu: True                               ==
==                 archi: liae-udt                           ==
==               ae_dims: 512                                ==
==                e_dims: 64                                 ==
==                d_dims: 64                                 ==
==           d_mask_dims: 32                                 ==
==       masked_training: True                               ==
==       eyes_mouth_prio: True                               ==
==           uniform_yaw: True                               ==
==         blur_out_mask: True                               ==
==             adabelief: True                               ==
==            lr_dropout: y                                  ==
==           random_warp: True                               ==
==      random_hsv_power: 0.0                                ==
==       true_face_power: 0.0                                ==
==      face_style_power: 0.0                                ==
==        bg_style_power: 0.0                                ==
==               ct_mode: none                               ==
==              clipgrad: False                              ==
==              pretrain: False                              ==
==       autobackup_hour: 0                                  ==
== write_preview_history: False                              ==
==           target_iter: 0                                  ==
==       random_src_flip: False                              ==
==       random_dst_flip: False                              ==
==            batch_size: 4                                  ==
==             gan_power: 0.0                                ==
==        gan_patch_size: 28                                 ==
==              gan_dims: 16                                 ==
==                                                           ==
==----------------------- Running On ------------------------==
==                                                           ==
==          Device index: 0                                  ==
==                  Name: NVIDIA GeForce RTX 3060 Laptop GPU ==
==                  VRAM: 3.39GB                             ==
==                                                           ==
===============================================================
Starting. Press "Enter" to stop training and save model.

i have waited here overnight to no avail

## Steps to reproduce

i have tried a multitude of different settings and followed both tutorials and had the same issue, i have re extracted deepfakelabs and made sure im using the right version 

## Other relevant information
windows and most recent python as of 5/3/2023

**Comment by JDTCC** [comment]
Did you resolve the issue yet?

**Comment by ghost** [comment]
Please close this issue.
